pkg/kubernetes/scripts/projects.js

```diff
@@ -103,6 +120,99 @@
         }
     ])
 
+
+    .controller('UserCtrl', [
+        '$scope',
+        '$routeParams',
+        '$location',
+        'kubeSelect',
+        'kubeLoader',
+        'projectData',
+        'projectActions',
+        'roleActions',
+        'ListingState',
+        function($scope, $routeParams, $location, select, loader, projectData, projectAction, roleActions, ListingState) {
+            loader.watch("users");
+            loader.watch("groups");
+            var user = $routeParams["user"] || "";
+            if (user) {
+
+                var userObj = select().kind("User").name(user);
+                if(!userObj || userObj.length < 1) {
+                    $location.path("/projects");
```

Comment:
I tried that, but due to delay in displaying users, sometimes a valid user is shown as invalid.I think this needs further design and better to come back to projects page.